### PR TITLE
Fix/dockerizing

### PR DESCRIPTION
**Pull request by elisehuard**
to debug deploy issues, I created a dev docker image we can also use for future docker-compose setups.

### CODE REVIEW

File: Dockerfile-dev [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+&& apt-get update \
+&& echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections \
+&& apt-get install -y \
+software-properties-common \
+oracle-java8-installer \
+oracle-java8-set-default
+
+ADD deploy-jars/bcpkix-jdk15on-1.55.jar /usr/lib/jvm/java-8-oracle/jre/lib/ext/bcpkix-jdk15on-1.55.jar
+ADD deploy-jars/bcprov-jdk15on-1.55.jar /usr/lib/jvm/java-8-oracle/jre/lib/ext/bcprov-jdk15on-1.55.jar
+ADD deploy-jars/java.security /etc/java-8-oracle/security/java.security
```
Comment (otfrom) [inline Dockerfile-dev]: I take it these 3 lines fix the crypto issues?

File: Dockerfile [hunk 2]
```diff
@@ -14,7 +14,8 @@ RUN add-apt-repository -y ppa:webupd8team/java \
 && echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections \
 && apt-get install -y \
 software-properties-common \
-oracle-java8-installer
+oracle-java8-installer \
+oracle-java8-set-default
```
Comment (otfrom) [inline Dockerfile]: I'm surprised that the Oracle JDK is giving us this problem (OpenJDK maybe, but not Oracle). 

Would moving to one of these docker images help at all? https://hub.docker.com/r/azul/zulu-openjdk/

File: Dockerfile-dev (hunk 1 above)
Comment (elisehuard) [inline Dockerfile-dev]: yes, exactly

File: Dockerfile (hunk 2 above)
Comment (elisehuard) [inline Dockerfile]: well, the problem occurred on Oracle, not OpenJDK :)

**Pull request merged**